### ISSUE TITLE
new Instagram API

### ISSUE THREAD

**Issue by sandros94** [atinux/nuxt-auth-utils#193]
So, when I implemented Instagram as a provider I wasn't aware of an [API change that was coming](https://developers.facebook.com/blog/post/2024/09/04/update-on-instagram-basic-display-api/?locale=en_US), in particular:

> Starting on December 4th, 2024, [Instagram Basic Display API](https://developers.facebook.com/docs/instagram-basic-display-api/) will no longer be available.
>
> [...]
> - After December 4th, 2024, there will no longer be a set of Instagram APIs for consumer developer apps
>
> [...]
> - Businesses and creators can leverage this new solution or use Instagram API with Facebook Login for their respective business use cases.

On one side this basically kills the project I was working on (🙄), but on the other side I need to update the implementation. So that if someone needs the "Instagram Professional" part they could still use it.

I didn't check what about Facebook's APIs, yet.

@atinux I'll be happy to track, study and work on this in the upcoming weeks. So if anything related comes up feel free to tag me.

**Comment by atinux** [comment]
Thank you for reporting it @sandros94 !

**Comment by kleinpetr** [comment]
does that mean the instgram provider will be out of service as of tomorrow?

**Comment by sandros94** [comment]
> does that mean the instgram provider will be out of service as of tomorrow?

yes, exactly. My original plan was to wait a little bit and hope for a change of mind, but then they did not as well as I got submerged by other work.

Soon I'll work on the updated instagram provider, but it will be limited to business accounts (afaik this should also apply to the facebook one, but not sure)
